app/Http/Controllers/Rdt/RdtEventParticipantListController.php

```diff
@@ -51,7 +51,9 @@ public function __invoke(Request $request, RdtEvent $rdtEvent)
             return RdtInvitationResource::collection($records->get());
         }
 
-        return RdtInvitationResource::collection($records->paginate($perPage));
+        $records = $perPage === null ? $records->paginate(count($records->get())) : $records->paginate($perPage);
```

Comment:
@rachadiannovansyah Kalau `$perPage === null`, kita ingin get semua record ya?

Berarti harusnya nggak perlu pakai `$records->paginate` kan ya?

```suggestion
        $records = $perPage === null ? $records->get() : $records->paginate($perPage);
```

Apakah karena kalau pakai `$records->paginate`, ada beberapa field terkait pagination yang diperlukan frontend?

Reply:
ahhhh iya betul, tidak digunakan juga, makasih kang @aldidoanta :100:
saya perbaiki dulu :+1:

Reply:
Sip mantap. Nanti jgn lupa dites ya. Kalo nggak salah, `$records->get()` sama `$records->paginate()` punya perbedaan struktur data di response-nya

Reply:
siapppp kang @aldidoanta :+1: